### PR TITLE
Store correct signal amount, store information on which checks failed for sensitivity bins

### PR DESCRIPTION
**Pull request by maxnoe**
This just adds a column on which of the 3 checks failed to the output

### CODE REVIEW

File: examples/calculate_eventdisplay_sensitivity_cut2.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+            energy_min=0.003 * u.TeV,
+            energy_max=330.00 * u.TeV,
+            spectral_index=-2.0,
+            max_impact=3000.00 * u.m,
+            viewcone=10.0 * u.deg)
+    },
+}
+
+
+def get_bg_cuts(cuts, alpha):
```
Comment (HealthyPear) [inline examples/calculate_eventdisplay_sensitivity_cut2.py:87]: Shouldn't this be aready in estimate_background?

File: examples/calculate_eventdisplay_sensitivity_cut2.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
+"""
+Example for using pyirf to calculate IRFS and sensitivity from EventDisplay DL2 fits
+files produced from the root output by this script:
+
```
Comment (HealthyPear) [inline examples/calculate_eventdisplay_sensitivity_cut2.py:4]: Probably better add a paragraph to explain the difference wrt to the main script

File: examples/calculate_eventdisplay_sensitivity_cut2.py [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+T_OBS = 50 * u.hour
+
+# scaling between on and off region.
+# Make off region 5 times larger than on region for better
+# background statistics
+ALPHA = 0.2
+
+# gh cut used for first calculation of the binned theta cuts
+INITIAL_GH_CUT = 0.0
```
Comment (HealthyPear) [inline examples/calculate_eventdisplay_sensitivity_cut2.py:46]: This shouldn't be used in this script

File: pyirf/tests/test_sensitivity.py [hunk 4]
```diff
@@ -27,6 +27,50 @@ def test_relative_sensitivity():
...
+    assert np.all(sensitivity['failed_checks'] == 0b100)
+
+    # not above 5 percent of remaining background
+    signal_hist['n_weighted'] = 699
+    bg_hist['n_weighted'] = 70_000
+    sensitivity = calculate_sensitivity(signal_hist, bg_hist, alpha=0.2)
+    assert len(sensitivity) == len(signal_hist)
+    assert np.all(np.isnan(sensitivity['relative_sensitivity']))
+    # too few signal
+    assert np.all(sensitivity['failed_checks'] == 0b010)
```
Comment (HealthyPear) [inline pyirf/tests/test_sensitivity.py:62]: is this hexadecimal?

**Review by HealthyPear — CHANGES_REQUESTED** [review]

File: examples/calculate_eventdisplay_sensitivity_cut2.py (hunk 1 above)
Comment (maxnoe) [reply examples/calculate_eventdisplay_sensitivity_cut2.py:87]: I didn't want to include this example add all, will remove

File: pyirf/tests/test_sensitivity.py (hunk 4 above)
Comment (maxnoe) [reply pyirf/tests/test_sensitivity.py:62]: No, `0b` is binary, `0x` is hex.

**Review by HealthyPear — APPROVED** [review]

**Pull request merged**